dora/core/client/fs/src/main/java/alluxio/client/file/PositionReadFileInStream.java

```diff
@@ -33,6 +38,7 @@
  */
 @NotThreadSafe
 public class PositionReadFileInStream extends FileInStream {
```

Comment:
The `PositionReadFileInStream` takes any implementation of `PositionReader`, which may or may not be a `NettyDataReader`. It might well be a `S3APositionReader`. Therefore, adding the "read from Alluxio" metric here may be inaccurate.

Reply:
The primary sources of "reading from Alluxio" are `NettyDataReader` and `LocalCachePositionReader` (the latter is a bit more troublesome as it has its own fallback-to-ufs logic).

Reply:
I will move the metrics into NettyReader